auth/auth.go

```diff
@@ -177,6 +177,16 @@ func (c *Client) CustomTokenWithClaims(uid string, devClaims map[string]interfac
 	return encodeToken(c.snr, defaultHeader(), payload)
 }
 
+// RevokeRefreshTokens revokes all refresh tokens for the specified user.
+// In addition to revoking all refresh tokens for a user, all ID tokens issued
+// before revocation will also be revoked at the Auth backend. Any request with an
+// ID token generated before revocation will be rejected with a token expired error.
+// Note that any tokens minted in the same epoch second as the revocation will still be valid.
+// If there is a chance that a token was minted in the last second, delay for 1 second before revoking.
+func (c *Client) RevokeRefreshTokens(ctx context.Context, uid string) error {
```

Comment:
Comment format for Go functions is something like:

```
// Single-lined overview of the function.
//
// Additional details as separate paragraphs.
```

Please format accordingly.

 I still think the race condition is an implementation detail that is not worth documenting. In fact it is going to confuse a lot of developers. In reality the chances of somebody running into this outside tests is almost non-existant.